nltk/text.py

```diff
@@ -410,20 +409,7 @@ def collocation_list(self, num=20, window_size=2):
         """
         Return collocations derived from the text, ignoring stopwords.
 
-            >>> from nltk.book import text4 # doctest: +NORMALIZE_WHITESPACE
-            *** Introductory Examples for the NLTK Book ***
-            Loading text1, ..., text9 and sent1, ..., sent9
-            Type the name of the text or sentence to view it.
-            Type: 'texts()' or 'sents()' to list the materials.
-            text1: Moby Dick by Herman Melville 1851
-            text2: Sense and Sensibility by Jane Austen 1811
-            text3: The Book of Genesis
-            text4: Inaugural Address Corpus
-            text5: Chat Corpus
-            text6: Monty Python and the Holy Grail
-            text7: Wall Street Journal
-            text8: Personals Corpus
-            text9: The Man Who Was Thursday by G . K . Chesterton 1908
+            >>> from nltk.book import text4
```

Comment:
This one is a bit tricky. `nltk.book` prints some stuff when imported, which is what we were testing for. However, it only prints it the first time it is imported.
if you do `pytest --doctest-modules nltk/text.py`, then it imports `nltk.book` for the first time here, and then it expects some printed output, but if you do `pytest --doctest-modules ./nltk`, then it imports it elsewhere differently (probably in some pytest code that traverses everything under `./nltk`), and then we expect this import to print nothing.

Long story short: the new situation passes on `pytest --doctest-modules nltk` (which is the important case), but fails on `pytest --doctest-modules nltk/text.py` (which is the unimportant case).